db/repos/rounds.js

```diff
@@ -170,6 +170,53 @@ class RoundsRepository {
 	restoreVotesSnapshot () {
 		return this.db.none(sql.restoreVotesSnapshot);
 	}
+
+	/**
+	 * Insert round information record to mem_rounds
+	 *
+	 * @param {string} address - Address of the account
+	 * @param {string} blockId - Block Id of hte during which the information updated
+	 * @param {Number} round - Round number during which update was made
+	 * @param {Number} amount - Amount updated during a round
+	 * @return {Promise}
+	 */
+	insertRoundInformationWithAmount (address, blockId, round, amount) {
+		return this.db.none(sql.insertRoundInformationWithAmount, {
+			address: address,
+			amount: amount,
+			blockId: blockId,
+			round: round
+		});
+	}
+
+	/**
+	 * Insert round information record to mem_rounds
+	 *
+	 * @param {string} address - Address of the account
+	 * @param {string} blockId - Block Id of hte during which the information updated
+	 * @param {Number} round - Round number during which update was made
+	 * @param {string} delegateId - Delegate Id for which to add round information
+	 * @param {string} mode - Possible values of '+' or '-' represents behaviour of adding or removing delegate
+	 * @return {Promise}
+	 */
+	insertRoundInformationWithDelegate (address, blockId, round, delegateId, mode) {
```

Comment:
This can be implemented with just a single line of code, and a single SQL file. A bit of an overkill having two separate methods and two separate SQL files for just a single `-` - minus sign.